src/ui/setup/index.ts

```diff
@@ -182,7 +182,14 @@ export async function bootstrapApp() {
     const userInfo = await getUserInfo();
     if (userInfo) {
       const recordingId = getRecordingId();
-      const rec = recordingId ? await getRecording(recordingId) : null;
+      let rec = null;
+      if (recordingId) {
+        try {
+          rec = await getRecording(recordingId);
+        } catch (e) {
+          console.warn("Error fetching recording");
```

Comment:
Why was this try/catch added? Is it related to the change you have in https://github.com/replayio/backend/pull/9450?

I think all of the places that call `getRecording` will now be try/catching it, except for the `RecordingCache`

Reply:
Pushed for the other case where it was not handled. But I don't think we go through this code path if we we fail to get the recording from one of earlier getRecording calls.